buku

```diff
@@ -31,6 +31,7 @@ import signal
 import logging
 import inspect
 import atexit
+import ruamel.yaml
```

Comment:
I think it can be handled easily without introducing a new lib and the parsing the input file line by line. Same goes for outfile as well.